environment.yml

```diff
@@ -7,11 +7,17 @@ dependencies:
     - python==3.8
     - numpy==1.19.1
     - matplotlib==3.3.0
-    - pytorch==1.5.0
+    - pytorch
```

Comment:
The pytorch_geometric installation has separate instructions in the README, since it requires some additional stuff. There's a specific note in the README indicating that the environment.yaml doesn't contain installation for pytorch_geometric.

Reply:
my bad, I didn't read that note. I installed pyg version 2.0.4 and py38_torch_1.11.0_cu115
Let it like that if you'd like to keep the same installation version.